lib/reporter/index.js

```diff
@@ -153,6 +157,47 @@ class Reporter extends SimplifiedReporter {
     this.testResults.logAssertion(result);
   }
 
+  /**
+   * @param {TheeNode} node
+   * @param {Object} result
+   */
+  logCommandResult(node, result) {
+
+    const isSuccess =  result === undefined ||
+      result == null ||
+      !((result instanceof Error) ||
+        (result.error instanceof Error) ||
+        result.status === -1
+      );
+
+    if (!this.shouldLogCommand(node)) {
+      this.testResults.logCommand({
+        name: node.fullName,
+        args: node.args,
+        startTime: node.startTime,
+        endTime: node.endTime,
+        elapsedTime: node.elapsedTime,
+        passed: isSuccess,
+        result
+      });
+    }
+  }
+
+  shouldLogCommand(node) {
+    if (node.parent.context && node.parent.context.isUserDefined) {
+      return true;
+    }
+
+    if (node && node.stackTrace) {
+      const invokationPath = node.stackTrace.split('\n')[1];
+
+      return invokationPath.includes(this.internalCommandPath) 
```

Comment:
@beatfactor To filter nested commands I have come up with this hack. Can I keep this or should we show all commands in the report?